tests/test_base.py

```diff
@@ -1687,6 +1691,340 @@ def test_kolmogorov_flux_scaling():
     print 'time for %s = %.2f'%(funcname(),t2-t1)
 
 
+def test_spergel():
+    """Test the generation of a specific Spergel profile using SBProfile against a known result.
+    """
+    import time
+    t1 = time.time()
+
+    for nu in test_spergel_nu:
+        filename = "spergel_nu{:.2f}.fits".format(nu)
+        savedImg = galsim.fits.read(os.path.join(imgdir, filename))
+        savedImg.setCenter(0,0)
+        dx = 0.2
+        myImg = galsim.ImageF(savedImg.bounds, scale=dx)
+        myImg.setCenter(0,0)
+
+        spergel = galsim.Spergel(nu=nu, half_light_radius=1.0)
+        # Reference images were made with old centering,
+        # which is equivalent to use_true_center=False.
+        myImg = spergel.draw(myImg, scale=dx, normalization="surface brightness",
+                             use_true_center=False)
+
+        np.testing.assert_array_almost_equal(
+            myImg.array, savedImg.array, 5,
+            err_msg="Using GSObject Spergel disagrees with expected result")
+
+        # Only nu >= -0.3 give reasonably sized FFTs
+        if nu >= -0.3:
+            do_kvalue(spergel, "Spergel(nu={:1}) ".format(nu))
+
+        # s.b. finite at origin iff nu > 0
+        # Consequently can only test sb sum and photon shooting for nu > 0.
+        if nu > 0.0:
+            np.testing.assert_almost_equal(
+                myImg.array.sum()*dx**2, myImg.added_flux, 5,
+                err_msg="Spergel profile GSObject::draw returned wrong added_flux")
+
+            # Test photon shooting.
```

Comment:
Sorry, I am a little confused by this restriction.  I thought the photon shooting used some approximations but still works for nu<0.  In this case, why shouldn't it be tested here?